packages/web-react/src/components/ButtonLink/__tests__/ButtonLink.test.tsx

```diff
@@ -12,6 +12,10 @@ import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
 import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import ButtonLink from '../ButtonLink';
 
+const buttonLinkDefaultProps = {
+  href: '#',
```

Comment:
**nitpick (if-minor):** Please, consider using `#section` or `#test`. The symbol `#` alone is not considered as valid href and I think that we should use valid URLs even in our tests and stories.